tensorflow_io/core/python/experimental/make_avro_record_dataset.py

```diff
@@ -79,14 +78,26 @@ def make_avro_record_dataset(
       prefetch_buffer_size: (Optional.) An int specifying the number of
         feature batches to prefetch for performance improvement.
         Defaults to auto-tune. Set to 0 to disable prefetching.
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
@ashahab Can you resolve the merge conflict here?

Reply:
@yongtang absolutely, sorry about that. @StanfordMCP

Reply:
@yongtang Fixed the issue, please take a look.